deploy/helm/sumologic/templates/setup/setup-clusterrole.yaml

```diff
@@ -4,7 +4,9 @@ kind: ClusterRole
 metadata:
   name:  {{ template "sumologic.metadata.name.setup.roles.clusterrole" . }}
   annotations:
-{{ toYaml .Values.sumologic.setup.clusterRole.annotations | indent 4 }}
+    helm.sh/hook: pre-install,pre-upgrade
+    helm.sh/hook-weight: "1"
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
```

Comment:
Can we move it to the templates? in case we want to change it?

Reply:
`weight` can be a parameter

Reply:
Done